server/svix-server/src/core/types.rs

```diff
@@ -135,6 +143,10 @@ macro_rules! json_wrapper {
             fn column_type() -> sea_orm::sea_query::ColumnType {
                 sea_orm::sea_query::ColumnType::JsonBinary
             }
+
+            fn array_type() -> sea_orm::sea_query::ArrayType {
+                sea_orm::sea_query::ArrayType::Json
```

Comment:
This doesn't look right. Should it be Json?

Reply:
I think so - sea_orm itself uses `ArrayType::Json` for the json Value type - which made me think that's what our json wrapper type should use as well.

I might just be wildly misunderstanding what this method is meant to do though 🤷

Reply:
Oh, I think then what you want is JsonBinary.

Essentially what it does (I think), is indicate how to tell PG to store this in an array. Since it's json we probably just want to use Json directly rather than a PG array of JSONs. Though use JsonBinary so that it matches the type we use for the column.

(unless there's no json binary, then json is fine)>

Reply:
Yeah there's no JsonBinary for ArrayType, unfortunately
https://docs.rs/sea-query/latest/sea_query/value/enum.ArrayType.html

Reply:
Weird, OK.